apps/server/src/modules/registration/domain/service/registration.service.ts

```diff
@@ -20,6 +36,26 @@ export class RegistrationService {
 		return registration;
 	}
 
+	public async completeRegistration(
+		registration: Registration,
+		language: LanguageType,
+		password: string
+	): Promise<void> {
+		const userDo = await this.createUser(registration, language);
+		if (userDo) {
```

Comment:
what do we do in the other case? seems like we should at least throw an error or something if this can happen

Reply:
fixed